aten/src/ATen/native/native_functions.yaml

```diff
@@ -187,6 +187,10 @@
   dispatch:
     CPU: _functional_assert_async_msg_cpu
 
+- func: _access_subclass_inner_tensor(Tensor src, str attr) -> Tensor
```

Comment:
It's not clear to me that we want this to be a dispatcher op (since it doesn't really need to go through the dispatcher at all). Any reason not to make it a special function that lives somewhere under `torch.export`?

Reply:
torch.export has a good ser/der support for aten ops, but i can do the special function way too